java/vector/src/test/java/org/apache/arrow/vector/TestListVector.java

```diff
@@ -633,11 +633,11 @@ public void testGetBufferAddress() throws Exception {
   public void testConsistentChildName() throws Exception {
     try (ListVector listVector = ListVector.empty("sourceVector", allocator)) {
       String emptyListStr = listVector.getField().toString();
-      assertTrue(emptyListStr.contains(ListVector.DATA_VECTOR_NAME));
+      Assert.assertTrue(emptyListStr.contains(ListVector.DATA_VECTOR_NAME));
 
-      listVector.addOrGetVector(FieldType.nullable(MinorType.INT.getType()));
+      listVector.addOrGetVector(FieldType.nullable(Types.MinorType.INT.getType()));
       String emptyVectorStr = listVector.getField().toString();
-      assertTrue(emptyVectorStr.contains(ListVector.DATA_VECTOR_NAME));
+      Assert.assertTrue(emptyVectorStr.contains(ListVector.DATA_VECTOR_NAME));
```

Comment:
The rest of the tests in this file match the way it was previously.  If you're going to change it here then I would prefer making the others consistent as well.

Reply:
done